authority-portal-frontend/src/app/core/services/config/app-config.ts

```diff
@@ -70,6 +71,7 @@ export interface AppConfigEnv {
   AUTHORITY_PORTAL_FRONTEND_BACKEND_URL: string;
   AUTHORITY_PORTAL_FRONTEND_LOGIN_URL: string;
   AUTHORITY_PORTAL_FRONTEND_LOGOUT_URL: string;
+  AUTHORITY_PORTAL_FRONTEND_UPDATE_PASSWORD_URL: string;
```

Comment:
Add documentation for this to the [user documentation](https://github.com/sovity/authority-portal/tree/main/docs/product/user-documentation), the [deployment README.md](https://github.com/sovity/authority-portal/blob/main/docs/deployment-guide/goals/production/README.md) and the [Deployment Migrations Notes-section of the CHANGELOG.md](https://github.com/sovity/authority-portal/blob/main/CHANGELOG.md#deployment-migration-notes)?